Feature_files/gmail.feature

```diff
@@ -0,0 +1,7 @@
+Feature: verify facebook page
+  @start
```

Comment:
Feature file contains same name of the feature and should contain description in next line.